migrations/migrate.sh

```
@@ -0,0 +1,2 @@
+liquibase changeLogSync --driver=org.mariadb.Driver --url="jdbc:mariadb://mariadb:3306/chat_users" --changeLogFile=/migrations/sql-migrations.yml --username=root --password=example
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)